client_app/src/VolunteerDashboard.js

```diff
@@ -0,0 +1,59 @@
+function VolunteerDashboard() {
+  return (
+    <div class="volunteer-dashboard">
+      <div class="column column-1">
+        <h3>Currently Registered</h3>
```

Comment:
This should have a list of real upcoming shifts (use the upcoming shifts component)